frontend/utils/auth.utils.ts

```
@@ -0,0 +1,54 @@
+const register = (username: string, email: string, password: string) => {
+  return api.post(
+    { username, email, password },
+    "/api/v1/accounts/users/register/"
+  );
+};
+
+const login = (email: string, password: string) => {
+  return api.post({ email, password }, "/api/v1/accounts/users/login/");
+};
+
+const logout = (token: string, logout_all: boolean = false) => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)